cloudbreak/config/config.go

```diff
@@ -121,38 +130,44 @@ func configRead(c *cli.Context) error {
 			set(fl.FlPassword.Name, config.Password)
 		}
 	}
-	if len(workspace) == 0 {
-		if len(config.Workspace) == 0 {
-			workspaceList := ws.GetWorkspaceList(c)
-			var workspaceID string
-			for _, workspace := range workspaceList {
-				if workspace.Name == c.String(fl.FlUsername.Name) {
-					workspaceID = strconv.FormatInt(workspace.ID, 10)
+	server = c.String(fl.FlServerOptional.Name)
+	username = c.String(fl.FlUsername.Name)
+	password = c.String(fl.FlPassword.Name)
+	if len(server) == 0 || len(username) == 0 || len(password) == 0 {
+		log.Error(fmt.Sprintf("configuration is not set, see: cb configure --help or provide the following flags: %v",
+			[]string{"--" + fl.FlServerOptional.Name, "--" + fl.FlUsername.Name, "--" + fl.FlPassword.Name}))
+		os.Exit(1)
```

Comment:
`panic` would be better, because easier to test and main catches it and calls `os.exit`